src/einsteinpy/utils/schwarzschild_utils.py

```diff
@@ -92,27 +117,29 @@ def metric(c, r, theta, Rs):
     return m
 
 
-def christoffels(c, r, theta, Rs):
+def christoffels(r, theta, M, c=constant.c.value, G=constant.G.value):
     """
     Returns the 3rd rank Tensor containing Christoffel Symbols for Schwarzschild Metric
 
     Parameters
     ----------
-    c : float
-        Speed of light
+    
     r : float
         Distance from the centre
     theta : float
         Angle from z-axis
-    Rs : float
-        Schwarzschild Radius
+    M : float
+        Mass of the massive body
+    c : float
+        Speed of light
 
     Returns
     -------
     ~numpy.array
         Numpy array of shape (4,4,4)
 
     """
+    Rs = utils.schwarzschild_radius_dimensionless(M, c, G)
```

Comment:
Same issue

Reply:
@shreyasbapat What I think let the utils import remain for the time being until `isort` problem is fixed. Because as soon as `from einsteinpy.utils import something, something, schwarzschild_radius_dimensionless` is written again `black`-`isort` conficts would emerge. Meanwhile #185 should be made urgent.

Reply:
Fixed in #205

Reply:
im on it.

Reply:
> im on it.

Rebase as #205 is merged. Kindly do the changes requested by @shreyasbapat and `black`+`isort` as the issue is fixed

Reply:
done